packages/reporter-vue/src/hooks/Hook.spec.ts

```diff
@@ -0,0 +1,283 @@
+// @ts-nocheck
+import Hook from './Hook.vue'
+import { Runnables } from '../store/reporter-store'
+import { h } from 'vue'
+
+const hooksRunnable = Runnables({
```

Comment:
I wonder if this can be more concise; the huge fixtures make it quite hard to read

Reply:
Yeah, it's terrible.